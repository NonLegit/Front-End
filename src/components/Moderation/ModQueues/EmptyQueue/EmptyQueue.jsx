import { Typography } from '@mui/material';
import {
  EmptyBox, Image,
} from './styles';
import Filter from '../Filter/Filter';

/**
 * if no posts are exist
 *
 * @component EmptyQueue
 * @returns {React.Component} EmptyQueue
 */

function EmptyQueue() {
  return (
    <>
      <Filter />
      <EmptyBox>
        <Image src="https://www.redditstatic.com/desktop2x/img/snoomoji/cat_blep.png" />
        <Typography variant="h6">The queue is clean!</Typography>
        <Typography variant="subtitle2" sx={{ color: '#7c7c7c' }}>Kitteh is pleased</Typography>
      </EmptyBox>
    </>
  );
}

export default EmptyQueue;
