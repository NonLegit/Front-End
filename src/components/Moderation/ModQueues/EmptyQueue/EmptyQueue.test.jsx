import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import EmptyQueue from './EmptyQueue';

test('Empty Queue test snapshot', async () => {
  const tree = renderer.create(
    <EmptyQueue />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
