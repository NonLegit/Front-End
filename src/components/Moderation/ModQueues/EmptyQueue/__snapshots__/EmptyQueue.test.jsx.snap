// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty Queue test snapshot 1`] = `
Array [
  <div
    className="MuiBox-root css-n276e7"
  >
    <div
      className="MuiBox-root css-k008qs"
    >
      <span
        className="MuiTypography-root MuiTypography-caption css-13fqsu0-MuiTypography-root"
      >
        Items 1-
        0
      </span>
      <span
        className="MuiTypography-root MuiTypography-caption css-13fqsu0-MuiTypography-root"
      >
        .
      </span>
      <span
        className="MuiTypography-root MuiTypography-caption css-13fqsu0-MuiTypography-root"
      >
        0
         
        selected
      </span>
    </div>
  </div>,
  <div
    className="MuiBox-root css-1g1ee7m"
  >
    <img
      className="css-1tcygg3"
      src="https://www.redditstatic.com/desktop2x/img/snoomoji/cat_blep.png"
    />
    <h6
      className="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
    >
      The queue is clean!
    </h6>
    <h6
      className="MuiTypography-root MuiTypography-subtitle2 css-fa75u-MuiTypography-root"
    >
      Kitteh is pleased
    </h6>
  </div>,
]
`;
